### ISSUE TITLE
New Chart Plugin

### ISSUE THREAD

**Issue by reach2rv** [ariacom/Seal-Report#4]
Hi,

Can we integrate highcharts.com charts. it would be great if you can suggest on which modules to work on to get in integrated.

Regards

Arvind

**Comment by ariacom** [comment]
Yes Highchart seems nice but it is not open source...
The effort to integrate a new charting tool should not be huge as it concerns mainly the presentation modules...with probably some small modifications in the .cs.

**Comment by reach2rv** [comment]
OK Can you please guide through which files needs modification so we can try to integrate and utilize for our own.

Regards,
Arvind

**Comment by ariacom** [comment]
Well, it would be mainly in Model View.cshtml where the JavaScript is injected in HTML...
If you master the Seal environment, I think the work can be done in 4-8 days.
Are you not happy with the NVD3 charts ?
https://plot.ly/javascript/ seems interesting, it worth a study at least.

**Comment by reach2rv** [comment]
Any update on this..

**Comment by ariacom** [comment]
yes, the 4.0 is being integrating, it includes plotly charts dn chartJS, hope we got a version in April

**Comment by reach2rv** [comment]
Thank you for bringing this latest changes.. I appreciate the efforts to implement Plotly, it gives perfect look more like a commercialized product. Many thanks again. I will test and report if anything needed.

**Comment by reach2rv** [comment]
http://echarts.baidu.com looks interesting for relatime updates between charts

**Comment by reach2rv** [comment]
Hi,
I want to get some customization done. Need to integrated  https://github.com/apexcharts/apexcharts.js. 
Please let me know the costing for the same. my email reach2rv"@"gmailcom

**Comment by weedkiller** [comment]
https://echarts.apache.org/examples/zh/index.html

**Comment by ariacom** [comment]
echarts seems a good candidate to be integrated in a new release...
